app/connectors/CompanyHouseConnector.scala

```diff
@@ -37,23 +37,22 @@ class CompanyHouseConnector @Inject()(appConfig: FrontendAppConfig, http: HttpCl
 
   def validateCRN(data: CompanyDetails)(implicit ec: ExecutionContext): Future[Option[Boolean]] = {
     implicit val hc: HeaderCarrier = HeaderCarrier().withExtraHeaders(("Authorization", appConfig.companyHouseRequestAuth))
-
     val httpClient = if (appConfig.proxyRequired) proxyHttp else http
 
-    httpClient.GET(appConfig.companyHouseRequestUrl + data.companyReferenceNumber)
+    httpClient.GET(appConfig.companyHouseRequestUrl + data.companyReferenceNumber, Seq(), Seq("Authorization" -> appConfig.companyHouseRequestAuth))
```

Comment:
I guess you could just name your header parameter and avoid passing the empty sequence.

```suggestion
    httpClient.GET(appConfig.companyHouseRequestUrl + data.companyReferenceNumber, headers = Seq("Authorization" -> appConfig.companyHouseRequestAuth))
```